packages/html-reporter/src/testFilesView.tsx

```diff
@@ -83,6 +84,7 @@ export const TestFilesHeader: React.FC<{
       <div data-testid='overall-duration' style={{ color: 'var(--color-fg-subtle)' }}>Total time: {msToString(report.duration ?? 0)}</div>
     </div>
     {metadataVisible && <MetadataView metadata={report.metadata}/>}
+    {report.title && <HeaderTitleView title={report.title} />}
```

Comment:
Why is header below the body?

Reply:
Discussed offline. Will modify in future PR.